samcli/commands/local/lib/swagger/parser.py

```diff
@@ -136,15 +136,17 @@ def get_authorizers(self, event_type: str = Route.API) -> Dict[str, Authorizer]:
                     auth_name,
                 )
 
-            enable_simple_response = authorizer_object.get(SwaggerParser._AUTHORIZER_SIMPLE_RESPONSES, False)
+            enable_simple_response = authorizer_object.get(SwaggerParser._AUTHORIZER_SIMPLE_RESPONSES)
             if event_type != Route.HTTP or payload_version != LambdaAuthorizer.PAYLOAD_V2 and enable_simple_response:
-                enable_simple_response = False
+                # only display message if simple response is defined
```

Comment:
Is this change for something unrelated to metrics?

Reply:
This appears to be old code from a completely different PR, will remove.